esp/templates/themes/editor.html

```diff
@@ -384,6 +409,13 @@ <h3 class="accordion-toggle" data-toggle="collapse" href="#advanced" rel="popove
                             <h3>Category: {{ category }}<hr/></h3>
                             <!-- Dropdown box for adding an optional variable override that is not yet defined -->
                             <div class="control-group opt_var_div hidden">
+                                <div> Accent 1 for each side tab refers to the background color of the heading bar and the top tabs when the specified number side tab is selected.
+                                </div>
+                                </br>
+                                <div>
+                                Accent 2 for each side tab refers to the link color in both the frame and the top tabs when the specified number side tab is selected. 
```

Comment:
I would call this the "text" color instead of the "link" color. It's also the text color in the side tab?